client/src/styles/Landing.css

```diff
@@ -0,0 +1,67 @@
+.container-main{
```

Comment:
Sorry I should have clarified this in the task, we want to keep all of the styling inside of our component files. Take a look at Material UI's function makeStyles, it shouldn't be complicated to refactor this into a makeStyles